package.json

```diff
@@ -76,6 +77,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "devDependencies": {}
+  }
```

Comment:
Not related to the course, but as you keep growing your app, is good practice to track which libraries are dev dependencies, so the bundled size of the app in production is optimized, give a look to [dev dependencies](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)